introduce the podcast (#1382)
oktadeveloper/okta-blog: _source/_posts/2023-06-08-identity-developer-podcast.md

---
layout: blog_post

> title: "Unlock identity expertise with our new developer podcast!"

Reviewer: ```suggestion
title: "Unlock Identity Expertise with Our New Developer Podcast!"
```

title casing

author: edunham
by: advocate

> communities: [devops]

Reviewer: I'd recommend adding other communities unless this is truly only devops based podcast

> communities: [devops]

Author: ```suggestion
communities: [devops,security,mobile,.net,java,javascript,go,php,python,ruby]
```

> communities: [podcast]

Author: ```suggestion
communities: [devops]
```

> description: ""

Reviewer: There should be a description as this feeds into snippet display on social (and SEO tags)

> description: ""

Author: ```suggestion
description: "Listen and learn about the nuances of Okta's identity ecosystem"
```

> tags: []

Reviewer: Recommend tags like "announcement, podcast, identity"

(podcast might be good because as you create podcast based posts, it will allow people to find them

> tags: []

Author: ```suggestion
tags: [announcement, podcast, identity, WebAuthN, passkeys, OIN, enterprise]
```

tweets:

> - ""

Reviewer: @isemona did you want us to add sample tweets for you?

- ""
- ""
image: blog/identity-developer-podcast/podcast.jpg 

> type: awareness|conversion

Reviewer: ```suggestion
type: awareness
```

---

Do you enjoy podcasts? I find that, like audiobooks, podcasts can transform boring tasks into an opportunity for entertainment and education. 

I'm delighted to announce a new offering from Okta's Workforce Developer Advocacy Team: our monthly podcast! 

> The audio version of the Workforce Identity Developer Podcast is available on [Casted](https://listen.casted.us/public/49/Workforce-Identity-Developer-Podcast-4ce90a5f), where you can also explore Auth0's ongoing [Authorization in software podcast](https://authorizationinsoftware.auth0.com/public/49/Authorization-in-Software-f9b69587) and [Identity Unlocked](https://identityunlocked.auth0.com/public/49/Identity%2C-Unlocked.--bed7fada) podcast as well.

Reviewer: I might consider adding some h2 and use some reinforcing keywords for better SEO here, such as (also a chance to use some of the other titles that we liked)

```
## Listen and learn with the workforce identity podcasts

The audio version...
```

> The audio version of the Workforce Identity Developer Podcast is available on [Casted](https://listen.casted.us/public/49/Workforce-Identity-Developer-Podcast-4ce90a5f), where you can also explore Auth0's ongoing [Authorization in software podcast](https://authorizationinsoftware.auth0.com/public/49/Authorization-in-Software-f9b69587) and [Identity Unlocked](https://identityunlocked.auth0.com/public/49/Identity%2C-Unlocked.--bed7fada) podcast as well.

Author: ```suggestion
## Listen and learn with the Workforce Identity Developer Podcast

The audio version of the Workforce Identity Developer Podcast is available on [Casted](https://listen.casted.us/public/49/Workforce-Identity-Developer-Podcast-4ce90a5f), where you can also explore Auth0's ongoing [Authorization in software podcast](https://authorizationinsoftware.auth0.com/public/49/Authorization-in-Software-f9b69587) and [Identity Unlocked](https://identityunlocked.auth0.com/public/49/Identity%2C-Unlocked.--bed7fada) podcast as well.
```

> In March, with product manager Megha Rastogi, we discussed what Workforce Identity means, and review some of the history of security that gotu s here. We dig into phishing resistant MFA, and compare the relative benefits and drawbacks of WebAuthN and Passkeys. 

Reviewer: ```suggestion
In March, with product manager Megha Rastogi, we discussed what Workforce Identity means and reviewed some of the history of security that got us here. We dig into phishing-resistant MFA and compare the relative benefits and drawbacks of WebAuthN and Passkeys. 
```

Reviewer: Grammarly and typos

> In March, with product manager Megha Rastogi, we discussed what Workforce Identity means, and review some of the history of security that gotu s here. We dig into phishing resistant MFA, and compare the relative benefits and drawbacks of WebAuthN and Passkeys. 

Reviewer: Another opportunity here for SEO to add a header about keywords with March's topic, same with April, and May

{% youtube <PiY5HDp0ABI> %} 

In April, with Director of Technical Strategy Matt Egan, we explored the [Okta Integration Network](https://www.okta.com/integrations/) and compared OAuth integration between applications to its predecessor, SSWS tokens. Spoiler alert: Handing out a token that confers a lot of perms indefinitely to anyone who holds it isn't a great idea these days! 

{% youtube <oHvukPsaAko> %}

> In May, with Product Acceleration Principal Architect Dan Marma, we discussed his recent Devday 2023 talk about [The Enterprise-Ready MVP](https://youtu.be/zL2U7qpdRZY), expanding our definition of "Okta Developers" to include everyone who builds code for customers who use Okta as their identity provider. Dan also shared some stories from the field, and I learned about some extreme use cases that enterprises have demanded from their identity providers and the apps that they use those identity providers to access! 

Reviewer: Is the branding supposed to be "Devday23" or is "Devday 2023" appropriate?

Author: I had no clue so I did it exactly as Auth0 did on the youtube playlist name for the videos

{% youtube <enWM5ZHnEp4> %}

> Every podcast has its own comments section on the (developer forum){https://devforum.okta.com/c/developer-podcast-comments/25}. Let us know what you think of each episode, and share any topics that you'd like to hear about in the coming months!

Reviewer: ```suggestion
You can also get alerted on new podcasts through our social channels! Follow us on [Twitter](https://twitter.com/oktadev) and subscribe to our [YouTube channel](https://www.youtube.com/c/OktaDev/) for new identity content. 

Every podcast has a comments section on the (developer forum){https://devforum.okta.com/c/developer-podcast-comments/25}. Let us know what you think of each episode, and share any topics that you'd like to hear about in the coming months!
```

author: edunham
by: advocate
communities: [devops,security,mobile,.net,java,javascript,go,php,python,ruby]
description: "Listen and learn about the nuances of Okta's identity ecosystem"
tags: [announcement, podcast, identity, WebAuthN, passkeys, OIN, enterprise]
tweets:
- ""
- ""
- ""
image: blog/identity-developer-podcast/podcast.jpg 
type: awareness
---

Do you enjoy podcasts? I find that, like audiobooks, podcasts can transform boring tasks into an opportunity for entertainment and education. 

I'm delighted to announce a new offering from Okta's Workforce Developer Advocacy Team: our monthly podcast! 

## Listen and learn with the Workforce Identity Developer Podcast

The audio version of the Workforce Identity Developer Podcast is available on [Casted](https://listen.casted.us/public/49/Workforce-Identity-Developer-Podcast-4ce90a5f), where you can also explore Auth0's ongoing [Authorization in software podcast](https://authorizationinsoftware.auth0.com/public/49/Authorization-in-Software-f9b69587) and [Identity Unlocked](https://identityunlocked.auth0.com/public/49/Identity%2C-Unlocked.--bed7fada) podcast as well.

> In March, with product manager Megha Rastogi, we discussed what Workforce Identity means and reviewed some of the history of security that got us here. We dig into phishing-resistant MFA and compare the relative benefits and drawbacks of WebAuthN and Passkeys. 

Author: ```suggestion

## WebAuthN & Passkeys

In March, with product manager Megha Rastogi, we discussed what Workforce Identity means and reviewed some of the history of security that got us here. We dig into phishing-resistant MFA and compare the relative benefits and drawbacks of WebAuthN and Passkeys. 
```

{% youtube <PiY5HDp0ABI> %} 

> In April, with Director of Technical Strategy Matt Egan, we explored the [Okta Integration Network](https://www.okta.com/integrations/) and compared OAuth integration between applications to its predecessor, SSWS tokens. Spoiler alert: Handing out a token that confers a lot of perms indefinitely to anyone who holds it isn't a great idea these days! 

Author: ```suggestion
## SSWS Auth Tokens vs OAuth

In April, with Director of Technical Strategy Matt Egan, we explored the [Okta Integration Network](https://www.okta.com/integrations/) and compared OAuth integration between applications to its predecessor, SSWS tokens. Spoiler alert: Handing out a token that confers a lot of perms indefinitely to anyone who holds it isn't a great idea these days! 
```

{% youtube <oHvukPsaAko> %}

> In May, with Product Acceleration Principal Architect Dan Marma, we discussed his recent Devday 2023 talk about [The Enterprise-Ready MVP](https://youtu.be/zL2U7qpdRZY), expanding our definition of "Okta Developers" to include everyone who builds code for customers who use Okta as their identity provider. Dan also shared some stories from the field, and I learned about some extreme use cases that enterprises have demanded from their identity providers and the apps that they use those identity providers to access! 

Author: ```suggestion
## Building SaaS for Enterprise Customers

In May, with Product Acceleration Principal Architect Dan Marma, we discussed his recent Devday 2023 talk about [The Enterprise-Ready MVP](https://youtu.be/zL2U7qpdRZY), expanding our definition of "Okta Developers" to include everyone who builds code for customers who use Okta as their identity provider. Dan also shared some stories from the field, and I learned about some extreme use cases that enterprises have demanded from their identity providers and the apps that they use those identity providers to access! 
```

{% youtube <enWM5ZHnEp4> %}

> You can also get alerted on new podcasts through our social channels! Follow us on [Twitter](https://twitter.com/oktadev) and subscribe to our [YouTube channel](https://www.youtube.com/c/OktaDev/) for new identity content. 

Author: ```suggestion
## What Next?

You can also get alerted on new podcasts through our social channels! Follow us on [Twitter](https://twitter.com/oktadev) and subscribe to our [YouTube channel](https://www.youtube.com/c/OktaDev/) for new identity content. 
```

---
layout: blog_post
title: "Unlock Identity Expertise with Our New Developer Podcast!"
author: edunham
by: advocate
communities: [devops,security,mobile,.net,java,javascript,go,php,python,ruby]
description: "Listen and learn about the nuances of Okta's identity ecosystem"

> tags: [announcement, podcast, identity, WebAuthN, passkeys, OIN, enterprise]

Author: ```suggestion
tags: [announcement, podcast, identity, webauthn, passkeys, oin, enterprise]
```

tweets:
- ""
- ""
- ""
image: blog/identity-developer-podcast/podcast.jpg 
type: awareness
---

Do you enjoy podcasts? I find that, like audiobooks, podcasts can transform boring tasks into an opportunity for entertainment and education. 

I'm delighted to announce a new offering from Okta's Workforce Developer Advocacy Team: our monthly podcast! 

## Listen and learn with the Workforce Identity Developer Podcast

The audio version of the Workforce Identity Developer Podcast is available on [Casted](https://listen.casted.us/public/49/Workforce-Identity-Developer-Podcast-4ce90a5f), where you can also explore Auth0's ongoing [Authorization in software podcast](https://authorizationinsoftware.auth0.com/public/49/Authorization-in-Software-f9b69587) and [Identity Unlocked](https://identityunlocked.auth0.com/public/49/Identity%2C-Unlocked.--bed7fada) podcast as well.

> ## WebAuthN & Passkeys

Reviewer: ```suggestion
## WebAuthN and Passkeys
```

Per our conventions to not use ampersand. But YAS! The header text is 🔥

Reviewer: Ah, and it looks like this fails the build, too because of the ampersand makes it an invalid tag

In March, with product manager Megha Rastogi, we discussed what Workforce Identity means and reviewed some of the history of security that got us here. We dig into phishing-resistant MFA and compare the relative benefits and drawbacks of WebAuthN and Passkeys. 

> {% youtube <PiY5HDp0ABI> %} 

Reviewer: ```suggestion
{% youtube PiY5HDp0ABI %} 
```

## SSWS Auth Tokens vs OAuth

In April, with Director of Technical Strategy Matt Egan, we explored the [Okta Integration Network](https://www.okta.com/integrations/) and compared OAuth integration between applications to its predecessor, SSWS tokens. Spoiler alert: Handing out a token that confers a lot of perms indefinitely to anyone who holds it isn't a great idea these days! 

> {% youtube <oHvukPsaAko> %}

Reviewer: ```suggestion
{% youtube oHvukPsaAko %}
```

> ## Building SaaS for Enterprise Customers

Reviewer: ```suggestion
## Building SaaS for enterprise customers
```

FYI, headers aren't title cased, so I think this is the only one to change

In May, with Product Acceleration Principal Architect Dan Marma, we discussed his recent Devday 2023 talk about [The Enterprise-Ready MVP](https://youtu.be/zL2U7qpdRZY), expanding our definition of "Okta Developers" to include everyone who builds code for customers who use Okta as their identity provider. Dan also shared some stories from the field, and I learned about some extreme use cases that enterprises have demanded from their identity providers and the apps that they use those identity providers to access! 

> {% youtube <enWM5ZHnEp4> %}

Reviewer: ```suggestion
{% youtube enWM5ZHnEp4 %}
```

> ## What Next?

Reviewer: ```suggestion
## What's next for identity education in the Okta developer podcasts?
```

grammar nit and trying to work in a couple more keywords (taking all the chances we can!). But having trouble figuring out a way to work in "identity" in there too so came up with this. definitely a suggestion though, so feel free to remove
